config/locales/en.yml

```diff
@@ -1889,6 +1889,10 @@ en:
       update_locale: Update settings related to your locale.
       upload_failed: Failed to upload
       you_admin_warning: You are currently an admin of this school. Please delete your admin access to enable account deletion .
+      weak: "Weak"
+      fair: "Fair"
+      medium: "Medium"
+      strong: "Strong"
```

Comment:
This repetition in `en.yml` makes me think that this value is probably something that can come from the `Zxcvbn` ReScript module.